src/processors/README.md

```diff
@@ -40,6 +40,20 @@ Example:
 
 No config needed.
 
+### filter-when-format
+
+`filter-when-format` will check if the first docblock comment of the file contains a `@format` tag. That means that the file was formatted with Prettier or a similar tool and that formatting-only rules (like `indent`) can be ignored for this file. The motivation for this processor is that ESLint rules sometimes have a different opinion about formatting than Prettier, so we want to conditionally turn them of and treat Prettier as the authority.
```

Comment:
How about this wording: 

`filter-when-format` will remove any break for the rules indicated if the first file docblock comment contains a `@format` tag, so these rule breaks will be filtered and won't be passed to the next processor or final report.

The original use case for this processor was trying to use a tool other than `ESLint` to format the code. Sometimes, `ESLint` had a different opinion about the final formatting , so we needed to conditionally turn some rules off if the file was already formatted (like the `indent` rule).